### ISSUE TITLE
Update badges on Readme

### ISSUE THREAD

**Issue by njbmartin** [chroma-sdk/Colore#252]

**Comment by Sharparam** [comment]
This will be fixed once `develop` is merged into `master`.

**Comment by njbmartin** [comment]
Could this repo move away from the current git flow? That way features and bugfixes make their way in more regularly.

**Comment by Sharparam** [comment]
Once 6.0 is out they can be made more regular. I've been away from home so haven't been able to do much work. There are a couple issues (#248, #168) we haven't heard from in a while.

@WolfspiritM was going to test out 6.0 as well to see if #168 is still relevant but haven't heard back from them.
